### ISSUE TITLE
autocorrelograms -- should not zero out the central bin

### ISSUE THREAD

**Issue by magland** [SpikeInterface/spikewidgets#52]
https://github.com/SpikeInterface/spikewidgets/blob/5d99c152f6499f50e4bd9756443e89c9c8bd5dfb/spikewidgets/widgets/correlogramswidget/correlograms_phy.py#L166

I do not think this line is correct:

```
# Remove ACG peaks.
correlograms[np.arange(n_clusters), np.arange(n_clusters), 0] = 0
```

Why should we zero out the central bin?

I do realize that phy does this. However, a different code does not:
https://github.com/int-brain-lab/ibllib/blob/master/brainbox/population/population.py

**Comment by samuelgarcia** [comment]
Hi Jeremy,
for the autocorrlogram, the central peak is each spike with itself. This leads to obvious very peaky at zeros plots that are useless, this is why I think that Cyril in phy removed the bin zeros.
I think I did the same in TDC.
This would be a mistake of course for the crosscorrelogram.
But it is not me that whote this. @alejoe91 @colehurwitz more info?
Sam

**Comment by alejoe91** [comment]
Hi @magland 
As @samuelgarcia said the central bin of the acg is useless and it's commonly removed. The code here is actually from Phy, which is quite fast

**Comment by magland** [comment]
I understand that the autocorrelogram should not include dt=0 correlations, but the central bin represents a range of dt correlations, and so the content is meaningful, even when dt=0 is excluded. I believe the code already excludes dt=0, and therefore that line can simply be excluded and you don't see an artificial peak at zero. Although it doesn't make a huge difference, I do think that the phy code is not quite correct in this regard.

**Comment by samuelgarcia** [comment]
if the code already exclude dt=0, then there is no reason.
Of course, but but for small bins, the difference should be very small (if sorting is good)

**Comment by magland** [comment]
I agree it doesn't make that much of a difference in practice, but I
believe the right thing in principle is to remove that line of code.
J
